src/SDK.ts

```diff
@@ -42,8 +42,9 @@ import { CustomContextData } from "./Utils/CustomContextData";
 
 export default class SDK implements ISDK {
   private static defaultConfiguration: ISDKConfiguration = {
-    getChatTokenRetryCount: 35,
+    getChatTokenRetryCount: 10,
```

Comment:
was this number as result of an agreement or part of any discussion ?

Reply:
Based on investigation, we will not reach 35 tries. We reduce the value as a preemptive measure.